envs/example/defaults-2.0.yml

```diff
@@ -638,3 +638,11 @@ serverspec:
 
 inspec:
   enabled: False
+
+#openstack-ansible-security overrides
```

Comment:
The role defaults come from openstack-ansible-security. I don't think we should change those